component/org.wso2.carbon.identity.oauth2.token.handler.clientauth.mutualtls/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/mutualtls/MutualTLSClientAuthenticator.java

```diff
@@ -209,9 +257,155 @@ protected boolean authenticate(X509Certificate registeredCert, X509Certificate r
         return trustedCert;
     }
 
+    /**
+     * Authenticate the client by comparing the attributes retrieved from the JWKS endpoint of the registered public
+     * certificate against the public key of the certificate presented at TLS hand shake for authentication.
+     *
+     * @param jwksUri     JWKS URI registered at service provider configuration.
+     * @param requestCert X.509 certificate presented to server during TLS hand shake.
+     * @return Whether the client was successfully authenticated or not.
+     */
+    private boolean authenticate(URL jwksUri, X509Certificate requestCert) throws OAuthClientAuthnException {
+
+        try {
+            return isAuthenticated(getResourceContent(jwksUri), requestCert);
+
```

Comment:
Better to remove this new line.

Reply:
Already FIxed: https://github.com/wso2-extensions/identity-oauth-addons/blob/4364f6d745b5ce7bfe6b7838dd8097a2cbcd3a3d/component/org.wso2.carbon.identity.oauth2.token.handler.clientauth.mutualtls/src/main/java/org/wso2/carbon/identity/oauth2/token/handler/clientauth/mutualtls/MutualTLSClientAuthenticator.java#L271